minecraft/protocol/item_stack.go

```diff
@@ -152,6 +152,8 @@ type StackResponseSlotInfo struct {
 	Count byte
 	// StackNetworkID is the network ID of the new stack at a specific slot.
 	StackNetworkID int32
+	// CustomName ...
```

Comment:
Documenting that this is the custom name of the new item stack and that it has something to do with text filtering here should be sufficient.